buildSrc/src/main/groovy/org/apache/geode/gradle/plugins/DependencyConstraints.groovy

```diff
@@ -101,6 +101,7 @@ class DependencyConstraints implements Plugin<Project> {
         api(group: 'com.nimbusds', name:'nimbus-jose-jwt', version:'8.11')
         api(group: 'com.sun.activation', name: 'javax.activation', version: '1.2.0')
         api(group: 'com.sun.istack', name: 'istack-commons-runtime', version: '3.0.11')
+        api(group: 'com.sun.mail', name: 'javax.mail', version: '1.6.2')
```

Comment:
Why is this removed?

Reply:
this addition is because we get javax.mail from two different places, due to a change in a Spring 5.3 dependency.  Without this, it brings in 1.6.1 and it looked funny to have both 1.6.1 and 1.6.2 in our license.

Reply:
this is intertwined with the activation issue, so GEODE-8066 will cover both